e2e/tests/ui-driven/src/helpers/addComponent.ts

```diff
@@ -1,9 +1,27 @@
 import { Locator, Page } from "@playwright/test";
 
-const createBaseInput = async (
+enum PlanXEditorComponent {
```

Comment:
I think it would be more maintainable to import the existing component type enum, and select by value in the select element here. There's already too many places where we need to update/sync this list so I'd be keen to avoid adding another where possible.

```ts
import { ComponentType } from "@opensystemslab/planx-core/types";

...

page.locator("select").selectOption({ value: type });
```

As a bonus, we could later make this an exhaustive switch which would force us to include a new test case here when adding a new component type.